src/Microsoft.Identity.Web/TokenCacheProviders/IMsalTokenCacheProvider .cs

```diff
@@ -22,6 +22,6 @@ public interface IMsalTokenCacheProvider
         /// Clear the cache
         /// </summary>
         /// <returns></returns>
-        Task ClearAsync();
+        Task ClearAsync(string clientId);
```

Comment:
Should we also update the comments or will there be a task to go over all of the comments in the project?